pep-0007.txt

```diff
@@ -58,6 +58,9 @@ C dialect
 
 * No compiler warnings with major compilers (gcc, VC++, a few others).
 
+* ``static inline`` functions should be preferred over macros in new
+  code (see PEP 670).
```

Comment:
Let's use the PEP role for autolinking:

```suggestion
  code (see :pep:`670`).
```

https://python.github.io/peps/pep-0012/#hyperlinks

Reply:
Not sure we should mention PEP 670 at all, given that the PEP as a whole has not yet been decided on.

Reply:
Ok ok, I removed it. The PEP is only half accepted :-p

Reply:
All right, LGTM! :)